woocommerce/woocommerce-android
# [Hazmat Shipping] Introduce Hazmat complete category selection UI

Summary
==========
Partially fix issue #9575 by creating the Hazmat category selection view with the expected Dialog detailing each category. This PR does not connect the selected Hazmat category with the Shipping Label creation yet. This will be introduced in the next PR. This one is just UI work.

Captures
==========
https://github.com/woocommerce/woocommerce-android/assets/5920403/a41a520e-a389-493f-bee2-68fb1670ba4b


How to Test
==========
1. Open the order details of an order eligible for Shipping label creation
2. Start the shipping label creation flow until you reach the Package details section
3. Inside the package details view, make sure the Contains hazardous materials toggle is visible and interactable
4. Make sure the toggle expands and collapses a basic view that will serve as the main control options for the HAZMAT declaration
5. Hit the `Hazardous material category` section, make sure it works as expected from a selection field

Update release notes:
